app/views/microposts/_form.html.erb

```diff
@@ -31,6 +36,11 @@
     <%= form.number_field :user_id %>
   </div>
 
+  <div class="field">
+    <%= form.label :category_id %>
+    <%= form.number_field :category_id %>
```

Comment:
Instead id, showing category's name makes UI better, I think 👍

Reply:
I wish i can show category's name, but i don't have enough time on that day :(